templates/src/csharp/build_packages_dotnetcli.bat.template

```diff
@@ -21,17 +21,21 @@
   set NUGET=C:\nuget\nuget.exe
   set DOTNET=dotnet
   
-  set -ex
```

Comment:
This actually wasn't doing anything, it's in a .bat file so it just prints "set -ex  Environment variable -ex not defined".
Behavior isn't changed by this edit.